Sources/JavaKitMacros/JavaFieldMacro.swift

```diff
@@ -79,4 +84,18 @@ extension JavaFieldMacro: AccessorMacro {
 
     return accessors
   }
+
+  private static func isInStaticContext(context: some MacroExpansionContext) -> Bool {
+    for lexicalContext in context.lexicalContext {
+      if let classSyntax = lexicalContext.as(ClassDeclSyntax.self) {
+        if classSyntax.name.trimmedDescription.starts(with: "JavaClass<") {
+          return true
+        } else {
+          return false
+        }
```

Comment:
that's just a `return classSyntax.name.trimmedDescription.starts(with: "JavaClass<")` right?